docs/additional/internal_aspects_en.md

```diff
@@ -0,0 +1,303 @@
+# Internal aspects
+
+This section describes aspects that must be adhered to by internal developers of Cloudogu GmbH.
+They contain requirements for quality assurance or, for example, basic definitions for a finished, final Dogu.
+Thus, this information is not necessarily relevant for external developers.
+However, the aspects can be used to define or improve quality standards.
+
+## Quality assurance
+
+Dogus can consist both of self-developed software and of encapsulated third-party applications.
+In the case of self-development, common quality assurance standards should apply, which are mentioned in the following
+sections below.
+When a Dogu encapsulates a third-party application, it is often not practical to test all of its features.
+The tests should then focus on the Dogu's added features, which typically represent the added benefit of a Dogu.
+
+### Quality assurance of the tool
+
+Quality assurance is in the responsibility of the owner of the software.
+For our software, we implement the following aspects for quality assurance:
+
+- Unit tests (80% code coverage)
+- Integration tests
+- End-to-end tests
+- Tools for static analysis e.g. Reviewdog, Linter (style, smells)
+- SonarQube (code coverage, smells, bugs, security vulnerabilities)
```

Comment:
SonarQube also offers static code analysis, otherwise things like code coverage, smells, bugs would not be possible. Should we join these two lines?

Reply:
This makes sense. I put both lines under `Static code analysis`